<h2>
Overview
</h2>

CardiacG2P is an evidence-based dataset for inherited cardiac condition
gene disease pairs. Each entry annotates a disease-associated gene with
information about the gene-disease relationship, including the
inheritance mode and allelic requirement, information pertaining to
disease mechanism (represented as a disease-associated variant
consequence), and known disease-relevant variant classes at a defined
locus.

This dataset is also available through G2P
<https://www.ebi.ac.uk/gene2phenotype>

<h2>
Citation
</h2>

*Josephs, K.S., Roberts, A.M., Theotokis, P. et al. Beyond gene-disease
validity: capturing structured data on inheritance, allelic requirement,
disease-relevant variant classes, and disease mechanism for inherited
cardiac conditions. Genome Med 15, 86 (2023).*
<https://doi.org/10.1186/s13073-023-01246-8>

<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSC2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
125645
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
DSC2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence;decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice\_region\_variant;splice\_acceptor\_variant;splice\_donor\_variant;frameshift\_variant;stop\_gained;missense\_variant;inframe\_deletion;inframe\_insertion;NMD\_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
17963498;33831308;24070718;17186466;19863551;24793512;20031616;21636032;23911551;17033975;34400560;31028357;23863954;31402444;26310507
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0012506
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
25.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
DSC3
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
3036
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

DSC2-related ARVC is due to decreased gene product level or altered gene
product sequence due to a variety of mechanisms (e.g. null alleles,
trafficking defects, impaired proteolytic processing, absence of or
impaired protein-protein interactions) (PMID: 31028357; 23911551;
NBK1131). Loss of function is the likely disease mechanism. ClinGen
found there was some evidence to support haploinsufficiency as a
mechanism. <https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3036>.
Autosomal dominant inheritance with incomplete penetrance is the most
common mode of transmission (PMID: 21636032; 33831308). Homozygous and
compound heterozygous variants have also been described in association
with ARVC with or without cutaneous features (PMID: 26310507; 23863954;
24793512; 24070718; 34400560). In some cases, these appear to reflect
autosomal recessive inheritance (PMID 24793512; 23863954, 33831308).
Instances of digenic inheritance have been identified with DSC2 variants
along with other desmosomal gene pathogenic variants (PMID: 24070718). A
number of DSC2 variants have been reported in the literature including
nonsense, frameshift, splice, missense and inframe insertions and
deletions (NBK1131; 17033975; 17963498; 17186466; 20031616; 19863551;
31402444). DSC2-related ARVC appears to be characterised by an increased
risk of biventricular involvement and heart failure when compared to
PKP2-related ARVC (PMID: 34400560).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSC2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
125645
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
DSC2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
biallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence;decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice\_region\_variant;splice\_acceptor\_variant;splice\_donor\_variant;frameshift\_variant;stop\_gained;missense\_variant;inframe\_deletion;inframe\_insertion;NMD\_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
17963498;33831308;24070718;17186466;19863551;24793512;20031616;21636032;23911551;17033975;34400560;31028357;23863954;31402444;26310507
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0012506
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
25.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
DSC3
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
3036
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

DSC2-related ARVC is due to decreased gene product level or altered gene
product sequence due to a variety of mechanisms (e.g. null alleles,
trafficking defects, impaired proteolytic processing, absence of or
impaired protein-protein interactions) (PMID: 31028357; 23911551;
NBK1131). Loss of function is the likely disease mechanism. ClinGen
found there was some evidence to support haploinsufficiency as a
mechanism. <https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3036>.
Autosomal dominant inheritance with incomplete penetrance is the most
common mode of transmission (PMID: 21636032; 33831308). Homozygous and
compound heterozygous variants have also been described in association
with ARVC with or without cutaneous features (PMID: 26310507; 23863954;
24793512; 24070718; 34400560). In some cases, these appear to reflect
autosomal recessive inheritance (PMID 24793512; 23863954, 33831308).
Instances of digenic inheritance have been identified with DSC2 variants
along with other desmosomal gene pathogenic variants (PMID: 24070718). A
number of DSC2 variants have been reported in the literature including
nonsense, frameshift, splice, missense and inframe insertions and
deletions (NBK1131; 17033975; 17963498; 17186466; 20031616; 19863551;
31402444). DSC2-related ARVC appears to be characterised by an increased
risk of biventricular involvement and heart failure when compared to
PKP2-related ARVC (PMID: 34400560).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
JUP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
173325
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Rare familial disorder with ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
JUP-related Naxos disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
strong
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
biallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
missense\_variant;inframe\_deletion;frameshift\_variant\_NMD\_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
25820315;21320868;25705887;17924338;10902626;11691526;20031617;21673311;32966140;8858175;20130592;28098346;15851108;8954745;31402444
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011017
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
CTNNG
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
6207
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

JUP-related Naxos disease (ARVC, woolly hair and palmoplantar
keratoderma) is due to altered gene product sequence causing loss of
function of JUP (PMID: 25705887; 21673311; 11691526; 10902626). JUP
encodes the protein plakoglobin. Kaplan et al found that in 4 Naxos
patients, Connexin43 expression at intercellular junctions was
significantly reduced and mutant plakoglobin was expressed but failed to
localize normally at intercellular junctions (<PMID:15851108>).
Inheritance is autosomal recessive. The initial nine patients described
ranged in age from 7 to 41 years. Since then, more patients have been
discovered carrying the disease with an estimate of 1:1000 in the
population of the Greek islands. The disease has also been diagnosed in
other countries (PMID: 32966140). A homozygous 2bp deletion in
plakoglobin (JUP), c.2157delTG, causing a frameshift and premature
termination of the protein and expression of a truncated plakoglobin
lacking 56 residues from the C terminus was described in 2000
(<PMID:10902626>). The truncated protein was identified on western blot.
In 2017, a homozygous missense variant was described in 7 unrelated
French-Canadian individuals. All had typical hair and skin findings; 4/7
had ARVC presenting after 28 years (PMID: 28098346). The effect of this
variant in the heterozygous state was not investigated. Two siblings of
consanguineous parents were found to have a homozygous 3bp deletion in
JUP c.901903delGAG (p.Glu301del). Both had woolly hair and skin
findings, only the older sister had ARVC and neither had palmoplantar
keratoderma (PMID: 28098346). In OMIM there have been reports of other
types of homozygous variants (nonsense, splice, missense) in JUP causing
overlapping phenotypes and segregating with disease. Data on biallelic
LoF variants are sparse. In mice, generation of a null mutation of the
plakoglobin gene by homologous recombination results in embryonic
lethality (PMID: 8954745; 8858175). There are 2 reports in humans who
had skin features but no obvious cardiomyopathy (PMID: 20130592;
21320868). In one, JUP expression in the skin was absent. Cardiac JUP
expression was not directly measured to establish the consequence in the
heart therefore it is not known whether variant allele was expressed,
degraded, or rescued by alternate splicing. To note dominant pathogenic
variants in JUP have also been rarely described in association with
ARVC. Asimaki et al reported a dominant variant in JUP in a German
family with ARVC and no obvious cutaneous abnormalities (PMID:
17924338). Other studies have identified heterozygous missense variants
however their pathogenicity is still debated (PMID 25820315; 20031617;
31402444).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence;decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Intronic variants with functional evidence:
NM\_000335.5(SCN5A):c.3228+551T&gt;G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice\_region\_variant;splice\_acceptor\_variant;splice\_donor\_variant;missense\_variant;inframe\_deletion;inframe\_insertion;stop\_gained\_NMD\_triggering;frameshift\_variant\_NMD\_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
32850980;9521325;20564468;32533946;20031634;17075016;17442746;25905440;11748104;20129283;29959160;33164571;33131149;25829473;30203441;32893267;29798782;39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
CMD1E
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
10593
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SCN5A-related Brugada syndrome is caused by decreased gene product level
or altered gene product sequence due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. To date, SCN5A is the only gene
classified as having definitive evidence as a cause of monogenic Brugada
syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A pathogenic variants
are identified in approx. 20- 30% of cases of European ancestry (PMID:
30139433; PMID 33164571). This contribution may differ in other
populations. SCN5A-related Brugada syndrome is characterized by
autosomal dominant inheritance with incomplete penetrance (PMID 9521325;
11748104; 25905440; 20031634; 33164571; NBK1517). Hundreds of variants,
both truncating and non-truncating, have been described in association
with Brugada syndrome. Kapplinger et al 2010 (PMID: 20129283) identified
293 distinct variants in SCN5A: 193 missense, 32 nonsense, 38
frameshift, 21 splice-site, and 9 inframe deletions/insertions. Walsh et
al 2020 (PMID: 32893267) found that non truncating variants were highly
enriched in European cases in the SCN5A transmembrane regions. More
recently, a rare non-coding variant in an SCN5A intronic enhancer region
has been identified in 9 patients with Brugada syndrome in Thailand
(3.9% of the patient cohort). The variant is highly enriched compared to
population matched controls and has been functionally validated. It
causes significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988). Gain of function variants are
associated with Long QT syndrome. Although loss of function appears to
be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNT2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
191045
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
TNNT2-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
missense\_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
11106718;29367541;27532257;15542288;20978592;20031601
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0007267
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
CMD1D;CMH2
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
11949
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

Pathogenic variants in TNNT2 cause DCM due to altered gene product
sequence. Only missense variants have confidently been reported as
pathogenic in TNNT2 DCM cases. There are no reports of truncating
variants and TNNT2 has a ClinGen Dosage haploinsufficiency score of 0,
and a PLi of 0 in gnomAD (19.6 expected and 20 observed). TNNT2-related
DCM is inherited in an autosomal dominant manner, and disease is often
an aggressive arrhythmogenic phenotype with high rates of heart failure
and sudden cardiac death. TNNT2 missense mutations account for 3% of
genetic DCM (<PMID:27532257>). TNNT2 variants are associated with an
early-onset and more severe form of DCM. Of note pathogenic variants in
TNNT2 are also associated hypertrophic and restrictive cardiomyopathies.
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TTN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
188840
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
TTN-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typified by incomplete penetrance; Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence;decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
Variants must occur in A-band or &gt;90% percentage spliced in (PSI)
exons. Pathogenic missense variants are very rare. Missense variants
with segregation evidence: NM\_001267550.2(TTN):c.2926T&gt;C
(p.Trp976Arg); NM\_001267550.2(TTN):c.533C&gt;A (p.Ala178Asp);
NM\_001267550.2(TTN):c.11674T&gt;A (p.Cys3892Ser)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice\_acceptor\_variant;splice\_donor\_variant;frameshift\_variant;stop\_gained;missense\_variant;stop\_gained\_NMD\_triggering;frameshift\_variant\_NMD\_triggering;exon\_loss\_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
23418287;9826585;9817758;25759365;19789381;25589632;32160020;31849696;18765796;28045975;27625337;11788824;37253077;29316444;27869827;29238064;26084686;32013205;26315439;22335739
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0011400
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
CMD1G
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
12403
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

Pathogenic variants in TTN cause an estimated 15-20% of familial DCM due
to decreased gene product level and altered gene product sequence. The
likely disease mechanism is loss of function however, it is unclear
whether this is due to haploinsufficiency or a dominant negative effect,
and it is likely that both mechanisms contribute. It is likely that the
reduced function has a direct effect on the sarcomere, leading to
impaired contractility. Truncating variants, specifically in exons
constitutively expressed in cardiac tissue (percentage spliced in (PSI)
&gt;90%) see (PMID: 25589632; 27869827; 32160020), are responsible for
the vast majority of TTN-related DCM cases. Missense variants are
difficult to interpret, and generally not classified as disease-causing,
although there are reports of at least three missense variants with
evidence of pathogenicity.TTN missense variants with segregation
evidence: p.Trp976Arg (PMID: 11788824) p.Ala178Asp (PMID: 27625337)
p.Cys3575Ser
(<https://www.biorxiv.org/content/10.1101/2020.09.05.282913v1.full.pdf>) -
note not yet peer reviewed. TTN-related DCM is inherited as an autosomal
dominant trait and displays incomplete and age-related onset. TTN
truncating variants are present in ~1% of the general population,
although these variants are more likely to reside in isoforms with lower
functional expression in cardiac tissue.
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ACTC1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
102540
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
ACTC1-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
missense\_variant;inframe\_deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
10966831;20031618;17611253;28972856;27532257;26061005;10330430;28007147;23604709
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0012799
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
10.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Cardiomyopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
ACTC
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
143
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

ACTC1 pathogenic variants cause HCM through altered gene product
sequence. The disease mechanism is not definitively known but may
involve impact on sarcomere force generation. There is currently little
evidence to support haploinsufficiency as a mechanism. Inheritance is
autosomal dominant. There is limited information regarding penetrance.
The initial missense variant identified by Mogensen et al was reported
to be highly penetrant in family members but with a variable age of
onset and severity (<PMID:10330430>). ACTC1 variants account for
&lt;3-5% of HCM cases (NBK1768; PMID: 20031618). Heterozygous missense
variants are the major type of pathogenic variants found. One inframe
deletion (a deletion of a single amino acid, Phe92del) has been reported
in association with HCM (PMID: 20031618). This variant is classified as
likely pathogenic on ClinVar. Liu et al suggest it may act to change the
local structure and arrangement of amino acids in the actomyosin binding
site (PMID: 28972856). Large deletions or duplications have not been
described.
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ACTN2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
102573
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
ACTN2-related cardiac and skeletal myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence;decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
frameshift\_variant\_NMD\_triggering;stop\_gained\_NMD\_triggering;missense\_variant;inframe\_deletion;deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
34802252;34526680;20022194;25224718;25173926;30630173;31956495;31680489;17097056;27532257;31983221;33500567;32973354;30701273;30900782;38311799;
39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0000591
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
ClinGen HCVD GCEP
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
164
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

Pathogenic variants in ACTN2 are definitively associated with cardiac
and skeletal myopathy. They can cause disease through decreased gene
product level and altered gene product sequence. ClinGen originally
curated ACTN2 for intrinsic cardiomyopathy (hypertrophic cardiomyopathy
(HCM) and dilated cardiomyopathy (DCM)) but this was re-evaluated and
expanded to include other cardiomyopathy types and skeletal muscle
phenotypes in December 2024 (PMID 39971408). ClinGen found no difference
in the mechanisms underlying these different disease entities and
therefore have grouped them together. Individuals can show variable
cardiomyopathy phenotypes, arrhythmias and sudden death (PMID:
34526680); the phenotype can vary between individuals in the same
family. Pathogenic variants in this gene can give rise to isolated left
ventricular hypertrophy (LVH). The inheritance pattern is autosomal
dominant; however there has been a report of a homozygous
ACTN2-truncating variant (in the terminal exon) with a recessive mode of
inheritance leading to restrictive cardiomyopathy (RCM) (PMID:
34802252). The skeletal phenotype is also variable and includes distal
myopathy and congenital myopathy. As per ClinGen ‘The skeletal muscle
cases reported in different families carrying ACTN2 variants had
homozygous and heterozygous missense variants reported. In the combined
cases, where patients presented with both cardiac and skeletal muscle
phenotypes, the proband showed frameshift and missense ACTN2 variants’
(PMID: 30701273, PMID: 30900782, PMID: 38311799). Missense, nonsense,
frameshift and larger deletions have all been reported in association
with cardiomyopathy phenotypes. 4 missense variants have segregation
data and 3 have functional data to support pathogenicity. Patients with
these variants presented with atypical and diverse phenotypes. In their
review, Walsh et al summarise what is known about ACTN2 variants and
their role in cardiomyopathy (PMID: 34526680). Missense variants with
segregation evidence: NM\_001103.4(ACTN2): c.355G&gt;A (p.Ala119Thr)
PMID: 20022194, <PMID:25224718>; NM\_001103.4(ACTN2): c.683T&gt;C
(p.Met228Thr) PMID: 25173926; NM\_001103.4(ACTN2): c.959T&gt;G
(p.Leu320Arg) PMID: 30630173; NM\_001103.4(ACTN2): c.1418A&gt;G
(p.Tyr473Cys) PMID: 31956495. Missense variants with functional
evidence: NM\_001103.4(ACTN2): c.740C&gt;T (p.Thr247Met) PMID: 31680489;
NM\_001103.4(ACTN2):c.332G&gt;T (p.Gly111Val) PMID: 17097056;
NM\_001103.4(ACTN2):c.355G&gt;A (p.Ala119Thr) PMID: 20022194. According
to Genome Aggregation Database (gnomAD), loss of function variants are
constrained in ACTN2. There have been reports of deletions in ACTN2
leading to disease. An exon 3-6 deletion leading to a frameshift was
found in two families with left ventricular non compaction (LVNC) and
HCM (PMID 32973354) and an exon 8-10 deletion leading to an in frame
deletion was found in a family with LVNC and arrhythmias (PMID:
34802252, PMID: 34526680). Truncating variants in ACTN2 have been
identified in approximately 1% of patients previously classified as
having LVNC. These variants were significantly enriched in individuals
with an LVNC classification compared to controls (0.65% vs. 0.01%, p =
1.3E-06) (PMID 33500567). Of note, it is unclear whether these ‘LVNC’
individuals exhibited both hypertrabeculation and cardiomyopathy or only
hypertrabeculation. Overall, rare variants are not enriched for ACTN2 in
HCM or DCM cohorts which suggests they are rarely causative of disease
so should be interpreted with caution in cardiomyopathy patients (PMID:
27532257, PMID: 31983221). Expert review panel: ClinGen Hereditary
Cardiovascular Disease Gene Curation Expert Panel. Review date:
17/03/2025
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CACNA1C
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
114205
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
CACNA1C-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
moderate
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
114180
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Long QT syndrome with an atypical presentation
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
CALM1-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
missense\_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
31170290;31983240;23388215;26969752
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0014548
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
CALML2
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
1442
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-26T11:53:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

CALM1-related LQTS is caused by an altered gene product sequence.
Reduction in the calcium dependent inactivation of the calcium channel
CaV1.2 results in increased inward calcium channel current (ICaL) and
repolarization delay. The disease mechanism is likely a dominant
negative effect. The three CALM genes encode an identical 149aa protein,
calmodulin. The CALM1 gene is located on chromosome 14, CALM2 on
chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is involved
in many calcium-dependent intracellular processes. All three CALM genes
have been classified as Definitive for LQTS and Moderate for CPVT,
certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable phenotypes. All
variants with evidence of pathogenicity identified in the CALM genes are
missense: at least 35 distinct missense variants have been identified
and reported in the International Calmodulinopathy Registry including 11
(31%) in CALM1, 16 (46%) in CALM2, and 8 (23%) in CALM3. The majority of
variants across the CALM genes affect amino acid residues in the EF-hand
Ca2+ binding loop III and IV. Some variants have additional evidence in
that the same substitution has been seen in paralogous CALM genes, or
that the same substitution has been seen in a related phenotype (LQTS).
Most variants are unique, but 9 were present in more than one index case
and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu,
identified in 10, 5, and 4 families, respectively.) appear to be
recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been
reported as associated with the LQTS phenotype, the variant p.Asn98Ser
has phenotypic variability, including LQTS, CPVT, idiopathic VF and
sudden death. The majority of CALM variants are de novo. Pathogenic
variants in the CALM genes have been associated with: presentation in
infancy or early childhood (up to 5 years); marked sinus bradycardia or
atrioventricular block and QT prolongation; and, predominantly with
CALM1 variants, a mild-to-severe neurological impairment, including
seizures, development delay, motor and/or cognitive disability. The
phenotype most frequently shown by patients with CALM variants is LQTS,
but some patients display other phenotypes including CPVT, idiopathic VF
and sudden unexplained death. Some variants have been associated with
both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that
‘despite the relatively small numbers of cases, a significant
association (p=0.001) was observed between location of mutation and
phenotype (Supplementary material online, Figure S1). Indeed, a
pathogenic variant in EF-hand IV Ca2+ binding loop was found in the
majority (17/32, 53%) of CALM-LQTS index cases but in only one of the
nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT
index cases were mostly located either in EF-hand III (n=5, 56%), or in
the inter-EF hand I-II linker (n=3, 33%).’
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNQ1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
607542
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Syndromic QT prolongation and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
KCNQ1-related Jervell and Lange-Nielsen syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
biallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
absent gene product;altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
splice\_acceptor\_variant;splice\_donor\_variant;frameshift\_variant;stop\_gained;missense\_variant;stop\_gained\_NMD\_triggering;frameshift\_variant\_NMD\_triggering;exon\_loss\_variant;complex\_structural\_alteration
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
27041150;23591039;9020846;27868350;23392653;31983240;11226272;25187895
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0024540
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
01.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
LQT;KCNA9
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
6294
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

Biallelic pathogenic variants in KCNQ1 cause JLNS due to absent gene
product level or altered gene product sequence. Both homozygous and
compound heterozygous pathogenic variants in KCNQ1 have been reported in
JLNS. Nonsense, frameshift, splice site, whole exon deletions and
missense variants have been reported as pathogenic. Bhuiyan and Wilde
(PMID: 23591039) compared two groups of patients with homozygous
variants in KCNQ1. Those patients where there was residual IKs (slowly
activating delayed rectifier potassium channel/current) even as little
as 10%, had QT prolongation but no hearing loss (autosomal recessive
LQTS), whereas those patients with complete absence of IKs had both QT
prolongation and hearing loss (JLNS). They concluded that homozygous or
compound heterozygous nonsense, frameshift or exon skipping variants
resulting in 100% loss of IKs would result in JLNS. In addition,
biallelic missense mutations that lead to a protein product that does
not traffic to the cell membrane (or is subject to nonsense mediated
decay) will lead to JLNS. Note: heterozygous variants in KCNQ1 lead to
dominant LQTS.
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNH2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene mim
</td>
<td style="text-align:left;">
152427
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Indication
</td>
<td style="text-align:left;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Referral indication
</td>
<td style="text-align:left;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease grouping
</td>
<td style="text-align:left;">
Classic SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; font-weight: bold;">
Disease name
</td>
<td style="text-align:left;font-weight: bold;">
KCNH2-related Short QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;">
definitive
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Allelic requirement
</td>
<td style="text-align:left;">
monoallelic\_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance
</td>
<td style="text-align:left;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Inheritance modifiers
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease-associated variant consequence
</td>
<td style="text-align:left;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mechanism
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
mutation consequence flag
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Variant classes reported with evidence of pathogenicity
</td>
<td style="text-align:left;">
missense\_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
pmids
</td>
<td style="text-align:left;">
29876509;21130771;30582453;30175559;29759541;29016797;9547387;18692916;14676148;25974115;7889573;30947366;30496390;19088443;30571592;7736582;29574456;21310316;15673388;15828882;31049424;28491588;25335996;19340359;31072576
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Disease accession
</td>
<td style="text-align:left;">
MONDO:0012312
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Curated date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel review date
</td>
<td style="text-align:left;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
Expert panel
</td>
<td style="text-align:left;">
Channelopathy expert panel
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
organ specificity list
</td>
<td style="text-align:left;">
Heart/Cardiovasculature/Lymphatic
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
panel
</td>
<td style="text-align:left;">
Cardiac
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
prev symbols
</td>
<td style="text-align:left;">
LQT2
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
hgnc id
</td>
<td style="text-align:left;">
6251
</td>
</tr>
<tr>
<td style="text-align:left;width: 6cm; ">
g2p gene disease pair entry date
</td>
<td style="text-align:left;">
2022-04-25T12:45:00Z
</td>
</tr>
</tbody>
</table>
<table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Narrative
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">

<br>

SQTS is a rare (&lt; 1/10,000) AD inherited arrhythmia syndrome
associated with atrial fibrillation, ventricular arrhythmia, and risk of
sudden cardiac arrest. Diagnosis is based on a diagnostic scorecard
(PMID: 21310316), similar to LQTS. Approximately 20% of diagnosed cases
will have a genetic cause, and KCNH2 is the most common identified
gene.It is noteworthy that of the 18 probands with SQTS in whom KCNH2
variants have been identified, 13 had one of 2 variants; 7 with
p.Thr618Ile variant (ClinVar Variation ID# 67297) and 6 with p.Asn588Lys
(ClinVar Variation ID# 14436; NM\_172056.2). There is high penetrance
for these recurrent variants. Experimental evidence derived from
non-patient cells, human-induced pluripotent stem cell-derived cells and
a rabbit animal model (PMID: 30496390) all support the relationship of
this gene with SQTS, with a gain-of-function mechanism. These
experimental studies demonstrate that genetic variants identified in
SQTS patients lead to potassium current perturbations concordant with
SQTS phenotype and shortening of the QT interval. Note loss-of-function
variants in KCNH2 are associated with Long QT Syndrome (LQTS).
</td>
</tr>
</tbody>
</table>
<br>
<p style="page-break-before: always;">
 
</p>

## Other potentially relevant variant classes

Which variant classes should I filter for? The main table shows variant
classes that have been reported to cause disease. We would expect other
variant classes with similar consequences to cause disease (e.g. if
NMD\_triggering frameshift cause disease, then NMD\_triggering nonsense
probably do too). This table below shows the full list of variant
classes mapped to each variant consequence.

<table class="table table-striped table-condensed" style="margin-left: auto; margin-right: auto;border-bottom: 0;">
<thead>
<tr>
<th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">

Disease-associated variant consequence

</th>
</tr>
<tr>
<th style="text-align:left;">
Decreased\_gene\_product\_level
</th>
<th style="text-align:left;">
Absent\_gene\_product\_level
</th>
<th style="text-align:left;">
Altered\_gene\_product\_sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
splice\_region\_variant
</td>
<td style="text-align:left;">
splice\_acceptor\_variant
</td>
<td style="text-align:left;">
splice\_acceptor\_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice\_acceptor\_variant
</td>
<td style="text-align:left;">
splice\_acceptor\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
splice\_acceptor\_variant\_NMD\_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice\_acceptor\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
splice\_donor\_variant
</td>
<td style="text-align:left;">
splice\_donor\_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice\_donor\_variant
</td>
<td style="text-align:left;">
splice\_donor\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
splice\_donor\_variant\_NMD\_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice\_donor\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
start\_lost
</td>
<td style="text-align:left;">
frameshift\_variant\_NMD\_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
start\_lost
</td>
<td style="text-align:left;">
frameshift\_variant
</td>
<td style="text-align:left;">
stop\_gained\_NMD\_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift\_variant
</td>
<td style="text-align:left;">
frameshift\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
stop\_lost
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift\_variant\_NMD\_triggering
</td>
<td style="text-align:left;">
stop\_gained
</td>
<td style="text-align:left;">
missense\_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
stop\_gained
</td>
<td style="text-align:left;">
stop\_gained\_NMD\_triggering
</td>
<td style="text-align:left;">
inframe\_insertion
</td>
</tr>
<tr>
<td style="text-align:left;">
stop\_gained\_NMD\_triggering
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
inframe\_deletion
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<span style="font-style: italic;text-decoration: underline;">Note:
</span>
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> Variant classes with a likelihood score of ‘3:possible’,
‘4:probable’ and ‘5:almost always’ for the different disease associated
variant consequences, see Tables 2 and 3 from Roberts et al.
</td>
</tr>
</tfoot>
</table>

*Roberts AM, DiStefano MT, Rooney Riggs E, Josephs KS, Alkuraya FS,
Amberger J, et al. Towards robust clinical genome interpretation:
developing a consistent terminology to characterize disease-gene
relationships - allelic requirement, inheritance modes and disease
mechanisms. MedRxiv. 2023;*
<https://doi.org/10.1101/2023.03.30.23287948>
